ecl/regress/regress.sh

```diff
@@ -113,7 +113,7 @@ fi
 
 if [[ $eclcc != '' ]]; then
     ## Set flags
-    default_flags="-P$target_dir -legacy -platform=thorlcr -fforceGenerate -fregressionTest -faddTimingToWorkunit=0 -b -S -shared"
```

Comment:
I don't think so - the timings are added to the workunit by default - this flag was a previous attempt to reduce the variation in the output.  This code wants to remove them completely, but also extract them to a separate file.  If you can thank of a use case which isn't a regression run then this could be added as a flag.
I should possibly remove the addTimingToWorkunit code since no longer useful.